cover/cover.go

```diff
@@ -159,9 +177,40 @@ func (fr *FileReport) IsNotCovered(row int) bool {
 	return false
 }
 
+// LocCovered returns the number of lines of code covered by tests
+func (fr *FileReport) LocCovered() (loc int) {
```

Comment:
Since we're exporting the Coverage field on the FileReport struct, let's unexport these functions.